### DISCUSSION TITLE
How to get owner record in create/update modal for relation resource?

### DISCUSSION THREAD

**Discussion by maherelgamil** [filamentphp/filament#3436] [Help] [answered]
How to get the owner record in creating/updating modal for relation resource?

**Comment by danharrin** [comment, marked as answer]
`$livewire->ownerRecord` or `$this->ownerRecord`

**Reply by maherelgamil** [reply]
@danharrin thanks

**Comment by MarJose123** [comment]
is this for the Relation resource? i can't use `$this` under the `RelationManager`

![image](https://user-images.githubusercontent.com/18107626/183581022-d4999bac-95e8-49ef-8370-bddbcbab4e06.png)

**Reply by danharrin** [reply]
`->default(fn ($livewire) => $livewire->ownerRecord->...)`

**Reply by MarJose123** [reply]
> 

I used this function during my trial and error

```php
->schema([
                Forms\Components\TextInput::make('Related_to')
                    ->default(function (HasRelationshipTable $livewire,){
                        return $livewire->ownerRecord->....;
                    })
                    ->maxLength(255),
            ]);
```

**Comment by Casmo** [comment]
For those coming from google: To get the record in the form() method itself:
```php
dd($form->getRecord());
```
